packages/core/package.json

```diff
@@ -24,6 +24,7 @@
   },
   "dependencies": {
     "@latitude-data/env": "workspace:^",
+    "@t3-oss/env-core": "^0.10.1",
```

Comment:
I have the perfect excuse for this sorry.

Reply:
plz... `if (process.env.BUILDING_CONTAINER == 'true') return`

Reply:
![image](https://github.com/user-attachments/assets/434d640e-3bfb-4015-b179-f8c05083737f)

Reply:
I don't understand why you don't want a tool [built for this propose](https://env.t3.gg/docs/introduction#support-for-multiple-environments)

Reply:
I actually like it for NextJS for the client-side safety measures that you mention ☝🏼 . I hate that we use it in api because it brings 0 value there and adding a dependency is just overhead.

Reply:
Also this package is type save. When you do `env.SOMETHING` you're sure that's a number or string. Doing the old way you still use untyped process.env